docs/development/getting_started.rst

```diff
@@ -23,7 +23,7 @@ Ubuntu/Debian
 
    sudo apt-get install -y dpkg-dev virtualbox-dkms linux-headers-$(uname -r) build-essential git
 
-We recommend using the latest stable version of Vagrant (``1.7.4`` at the time
+We recommend using the latest stable version of Vagrant (``1.8.4`` at the time
```

Comment:
Not actually running 1.8.5 over here, since it hasn't landed in the Debian Stretch repos yet. The [changelog](https://github.com/mitchellh/vagrant/blob/9c299a2a357fcf87f356bb9d56e18a037a53d138/CHANGELOG.md#185-july-18-2016) shows a number of bug fixes, so let me upgrade via the deb package and run through testing the SD VMs before recommending it blindly.